templates/gitea.ini.j2

```diff
@@ -132,27 +113,23 @@ SSL_MODE = {{ gitea_db_ssl }}
 PATH = {{ gitea_db_path }}
 LOG_SQL = {{ gitea_db_log_sql | ternary('true', 'false') }}
 {{ gitea_database_extra_config }}
-;
-;
 ; -> https://docs.gitea.com/next/administration/config-cheat-sheet/#indexer-indexer
 [indexer]
 ISSUE_INDEXER_PATH = {{ gitea_home }}/indexers/issues.bleve
+ISSUE_INDEXER_TYPE = {{ gitea_issue_indexer_type }}
+ISSUE_INDEXER_CONN_STR = {{ gitea_issue_indexer_conn_str }}
```

Comment:
Are these variables somewhere defined?

gitea_issue_indexer_type
gitea_issue_indexer_conn_str

Reply:
I'll do it a new PR.